galaxykubeman/templates/rbac.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.rbac }}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ include "galaxykubeman.roleName" . }}
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: ["", "extensions", "apps"]
+    resources: ["*"]
+    verbs: ["*"]
+  - apiGroups: ["batch"]
+    resources:
+      - jobs
+      - cronjobs
+    verbs: ["*"]
+
+---
+
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ .Release.Namespace }}-rbac-binding
```

Comment:
Could use `{{ include "galaxykubeman.roleName" . }}-binding` here to use the same base of the role name